src/index.js

```diff
@@ -1,5 +1,49 @@
-function helloWorld(name) {
-    return "Hello World " + name;
+/**
```

Comment:
We can specify to exclude specific files from the coverage analysis by adding it to this file: Eg: https://github.com/aicore/Core-Analytics-Server/blob/main/.nycrc.unit.json#L17